packages/components/checkbox/src/checkbox-group.tsx

```diff
@@ -26,7 +26,7 @@ const CheckboxGroup = forwardRef<"div", CheckboxGroupProps>((props, ref) => {
         <CheckboxGroupProvider value={context}>{children}</CheckboxGroupProvider>
       </div>
       {errorMessage ? (
-        <div {...getErrorMessageProps()}>{errorMessage}</div>
```

Comment:
@wingkwong why do we need to explicitly set the type? this should come from `@react-types/shared`

Reply:
`UseCheckboxGroupProps` -> `AriaCheckboxGroupProps` -> `CheckboxGroupProps` -> `HelpTextProps`

```ts
export interface HelpTextProps {
  /** A description for the field. Provides a hint such as specific requirements for what to choose. */
  description?: ReactNode,
  /** An error message for the field. */
  errorMessage?: ReactNode | ((v: ValidationResult) => ReactNode)
}
```

setting explicitly since we don't need to use it as a function

Reply:
@wingkwong  I think we will need to support the function one once @ryo-manba finishes the react-aria migration